packages/compass-web/src/index.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { screen, render, cleanup, waitFor } from '@testing-library/react';
+import type { ConnectionOptions } from 'mongodb-data-service';
+import { expect } from 'chai';
+import { CompassWeb } from './';
+import Sinon from 'sinon';
+import EventEmitter from 'events';
+import ConnectionString from 'mongodb-connection-string-url';
+
+function mockDb(name: string) {
+  return { _id: name, name };
+}
+
+class MockDataService extends EventEmitter {
```

Comment:
Now we know the bare minimum of commands that data service needs to implement somewhat for compass to work 😅